### ISSUE TITLE
Consultation on raw dataset issues

### ISSUE THREAD

**Issue by ArcueidXue** [IntelligentRobotLearning/pseudocode_learning_in_simulation#8]
**Consultation on raw dataset issues**

Thank you very much for your work. Your work is innovative and inspiring, and find the content described in your papers valuable for further research.

Our lab is also conducting research on flexible exoskeleton robots, and we would like to know whether your IMU nine-axis data undergoes any special filtering or other processing before being fed into the neural network model for training? As well as the type of IMU data fed into the neural network are those, and how many dimensions are there? This will help us to further process and explore the research direction of the paper.

If possible, I would very much like you to publicize some of the raw datasets of the experiments conducted in your paper, which would be very helpful for further research on flexible exoskeleton robots.
